ui/src/literature/components/CitationsLinkAction.jsx

```diff
@@ -1,30 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { LoginOutlined } from '@ant-design/icons';
 
-import { Link } from 'react-router-dom';
-import { LITERATURE } from '../../common/routes';
 import ListItemAction from '../../common/components/ListItemAction';
 import EventTracker from '../../common/components/EventTracker';
-import FormattedNumber from '../../common/components/FormattedNumber';
-import IconText from '../../common/components/IconText';
-import pluralizeUnlessSingle from '../../common/utils';
+import IncomingLiteratureReferencesLinkAction from '../../common/components/IncomingLiteratureReferencesLinkAction';
 
 function CitationsLinkAction({ recordId, citationCount, trackerEventId }) {
   return (
     <ListItemAction>
       <EventTracker eventId={trackerEventId}>
-        <Link to={`${LITERATURE}?q=refersto:recid:${recordId}`}>
-          <IconText
-            text={
-              <>
-                <FormattedNumber>{citationCount}</FormattedNumber>{' '}
-                {pluralizeUnlessSingle('citation', citationCount)}
-              </>
-            }
-            icon={<LoginOutlined />}
-          />
-        </Link>
+        <IncomingLiteratureReferencesLinkAction
```

Comment:
you should move the ```EventTracker``` and the ```ListItemAction``` inside the ```IncomingLiteratureReferencesLinkAction```. Even though ```EventTracker``` wasn't added in all the places you are using this, it would be good to add it.